### DISCUSSION TITLE
Building hyprland with wlroots release

### DISCUSSION THREAD

**Discussion by b1rger** [hyprwm/Hyprland#1258] [Q&A] [answered]
Hi,

I'm thinking about packaging hyprland for Debian. At the moment, hyprland uses an unreleased version of wlroots. Is this only a temporary workaround and will hyprland switch back to a wlroots release at some point? Embedded code copies in Debian are only allowed in exceptional cases, so having a custom copy of an unreleased library as part of a package will not be possible. If hyprland plans to switch back to use a released version of wlroots, I could upload a packaged version.

**Comment by vaxerski** [comment, marked as answer]
See #302.

We have no plans to use tagged wlroots. You can statically link against wlroots if building with meson, although that of course will still require pretty recent releases of wayland-protocols, libwayland and (usually the number one offender) libdrm.
